realtime/server.js

```diff
@@ -1,14 +1,16 @@
-var io = require('socket.io');
-var req = require('request');
-var fs = require('fs');
-var winston = require('winston');
+var io = require('socket.io'); //include websockets
+var req = require('request'); //include request(supports http)
+var fs = require('fs'); //include file system
```

Comment:
Assume your readers know node.js, therefore they know what `require` does as well as the node.js standard library. This comment is unnecessary.